kythe/cxx/verifier/verifier.h

```diff
@@ -201,6 +201,9 @@ class Verifier {
   /// \brief Look for assertions in file node text.
   void UseFileNodes() { assertions_from_file_nodes_ = true; }
 
+  /// \brief Only raise a warning if a file VName is missing.
+  void WarnOnMissingFileVNames() { warn_on_missing_file_vnames_ = true; }
```

Comment:
`AllowMissingFileVNames` ? something that indicates these will no longer be an error